packages/playwright/src/runner/rebase.ts

```diff
@@ -143,3 +149,40 @@ function findGitRoot(dir: string, cache: Map<string, string | null>): string | n
   cache.set(dir, parentResult);
   return parentResult;
 }
+
+function applyPatchWithConflictMarkers(oldText: string, newText: string) {
+  const diffResult = diff.diffLines(oldText, newText);
+
+  let result = '';
+  let conflict = false;
+
+  diffResult.forEach(part => {
+    if (part.added) {
+      if (conflict) {
+        result += part.value;
+        result += '>>>>>>> SNAPSHOT\n';
+        conflict = false;
+      } else {
+        result += '<<<<<<< HEAD\n';
+        result += part.value;
+        result += '=======\n';
+        conflict = true;
+      }
+    } else if (part.removed) {
+      result += '<<<<<<< HEAD\n';
+      result += part.value;
```

Comment:
Why is pure remove a conflict?

Reply:
everything is shown as a 3 way conflict in this mode